go.mod

```diff
@@ -1,36 +1,38 @@
 module knative.dev/func
 
-go 1.21
+go 1.22
```

Comment:
Do we really need to jump on 1.22? Is it convenience or forced by the dependencies?

Reply:
`go mod tidy` did this.

Reply:
Well, it's min version of required Go APIs used, not necessarily go-toolchain used for building it. It may create a downstream issues, because of slow Golang updates in distros like RHEL.

Reply:
> `go mod tidy` did this.

`go mod tidy` is accurate to bump min go, usually. I guess some of the modules now declares itself as `1.22` then.

Reply:
@dsimansk also I cannot even downgrade it manually, I believe `./hack/verify-codegen.sh` would failed.